ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/Metrics.scala

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.v1
```

Comment:
We shouldn't use `participant-state` as a convenient place to share code. This is the main API package for ledger backends. `Metrics` depending on dropwizard-metrics doesn't really fit here.

Reply:
What about adding all the metrics related support into a separate Bazel module?

Reply:
Moved to _participant-state-metrics_.